django/http/response.py

```diff
@@ -205,7 +205,8 @@ def set_signed_cookie(self, key, value, salt='', **kwargs):
         return self.set_cookie(key, value, **kwargs)
 
     def delete_cookie(self, key, path='/', domain=None):
-        self.set_cookie(key, max_age=0, path=path, domain=domain,
+        secure = True if key.startswith('__Secure-') or key.startswith('__Host-') else None
```

Comment:
No comment on the overall change, but this line can be simplified to:

```py
secure = key.startswith(('__Secure-', '__Host-'))
```

[`str.startswith()`](https://docs.python.org/3/library/stdtypes.html#str.startswith) accepts a tuple of values.